squad/core/history.py

```diff
@@ -9,6 +9,7 @@
 class TestResult(object):
 
     def __init__(self, test):
+        self.known_issue = test.known_issue
```

Comment:
what happens if there is more than one active KnownIssue for a test?

Reply:
good question. right now the code assumes there is only one, which is clearly wrong. I will have to rework this.